plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/JavaPackageValidator.java

```diff
@@ -16,15 +16,15 @@
   public static IStatus validate(String packageName) {
     
     if (packageName == null) {
-      return new Status(1, PLUGIN_ID, 45, "null package name", null);
+      return new Status(IStatus.ERROR, PLUGIN_ID, 45, "null package name", null);
```

Comment:
I made them up, no particular reason except that if one shows up I'd like to be able to find the relevant section of code easily. Now that I think about I should pick a value even less likely to appear by coincidence such as 9423757 or 1543234